frontstage/controllers/eq_payload.py

```diff
@@ -0,0 +1,120 @@
+import logging
```

Comment:
Doesn't seem right to me that this mixed in with controllers as it's more of a model if we're still using the MVC analogy.  I've seen other places put it in common/ but I don't know if that's right either...
I won't hold up the PR on this, but if you feel the same way then maybe move it somewhere more sensible?